internal/adapters/clients/github.go

```diff
@@ -266,20 +255,20 @@ func (gc *GithubClient) GetCommitCode(context context.Context, owner, repo, comm
 // Returns:
 //   - The created PullRequestComment object.
 //   - An error if any occurred during the API request.
-func (gc *GithubClient) AddCommentToPullRequestReview(ctx context.Context, owner, repo string, prNumber int, commentBody, commitID, path string, startLine, line int) (*github.PullRequestComment, error) {
+func (gc *GithubClient) AddCommentToPullRequestReview(ctx context.Context, owner, repo string, prNumber int, commentBody, commitID, path string, startLine, line int, startSide, side Side) (*github.PullRequestComment, error) {
 	comment := &github.PullRequestComment{
 		// Text content of the comment
 		Body: github.String(commentBody),
 		// SHA of the commit to comment on
 		CommitID: github.String(commitID),
 		// Filepath which the comment applies
 		Path: github.String(path),
-		// Position in the diff where the comment should be applied
-		// Position: github.Int(position),
 		// First line of range you want to comment on
 		StartLine: github.Int(startLine),
 		// Last line of range you want to comment on
-		Line: github.Int(line),
+		Line:      github.Int(line),
+		Side:      github.String(string(side)),
```

Comment:
Good addition of SubjectType to clarify the type of the subject for the PR comment. This makes the comment context clearer in API requests.